src/main/java/io/jenkins/plugins/evergreen/EvergreenUpdateLevelPageDecorator.java

```diff
@@ -0,0 +1,48 @@
+package io.jenkins.plugins.evergreen;
+
+import com.google.common.annotations.VisibleForTesting;
+import hudson.Extension;
+import hudson.model.PageDecorator;
+import net.sf.json.JSONException;
+import net.sf.json.JSONObject;
+import net.sf.json.groovy.JsonSlurper;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.logging.Logger;
+
+
+@Extension
+public class EvergreenUpdateLevelPageDecorator extends PageDecorator {
+
+    private static final Logger LOGGER = Logger.getLogger(EvergreenUpdateLevelPageDecorator.class.getName());
+    private File manifest;
+
+    @VisibleForTesting
+    EvergreenUpdateLevelPageDecorator(File manifestFilePath) {
+        this.manifest = manifestFilePath;
+    }
+
+    public EvergreenUpdateLevelPageDecorator() {
+        final String evergreenData = System.getenv("EVERGREEN_DATA");
+        manifest = new File(evergreenData, "updates.json");
+    }
+
+    public String getUpdateLevel() {
+        String updateLevel = "N/A";
+
+        if (manifest.exists()) {
+            try {
+                final JSONObject parsedJson = (JSONObject) new JsonSlurper().parse(manifest);
+                updateLevel = "" + ((JSONObject) parsedJson.get("meta")).get("level");
+            } catch (IOException e) {
+                updateLevel = "Unable to parse: " + e.getMessage();
```

Comment:
If the exception is thrown, then the displayed message in the footer would be the error message. Is that what we actualy want in that case? I think it might be weird to see the an error in the footer. Making use of @alecharp's idea, the administrative monitor could handle both, the error and the "N/A" case.
OTOH, many administrators don't take care of the monitors, so the monitor could be ignored. Even so, I think it could be better approach from the point of view of the UI.

Reply:
I agree with @fcojfernandez here.

Reply:
Well yes, that's what I wanted this to do, though TBH if there's no manifest it's likely *everything* is terribly broken.
An admin monitor for this is an idea, though for now my goal was to add a feature, not handle all exception cases.

Reply:
I filed https://issues.jenkins-ci.org/browse/JENKINS-54344 to followup on the AdminMonitor idea,which I also think makes sense. :)